server/lib/base_konnector.js

```diff
@@ -2,6 +2,7 @@ const _ = require('lodash')
 const printit = require('printit')
 const slugify = require('cozy-slug')
 const fetcher = require('./fetcher')
+const package = require('../../package.json')
```

Comment:
Ths path is also wrong. should be ../../../package.json (as the target file will be in build/server/lib...